keras/layers/convolutional_test.py

```diff
@@ -162,6 +162,16 @@ def fn(inpt):
       self.assertEqual(outp1_shape, layer(inpt1).shape)
 
 
+def test_conv1d_invalid_output_shapes(self):
+    kwargs = {'filters': 2, 'kernel_size': 20}
+    with self.assertRaisesRegex(ValueError, r"""
```

Comment:
Please update error message here. You just need to check for the start of the string, not the entire message

Reply:
(Note that I've updated the error that was getting raised)